packages/cysync-core/src/dialogs/Inheritance/SilverPlanPurchase/context/index.tsx

```diff
@@ -74,9 +75,10 @@ export const InheritanceSilverPlanPurchaseDialogProvider: FC<
       ...wallets.map(e => ({
         ...e,
         isDeleted: deletedWalletIds.includes(e.__id),
+        isActive: Boolean(plans.findIndex(p => p.walletId === e.__id) !== -1),
```

Comment:
We don't want to include all plans, you should consider the `purchasedAt` and `expireAt` values.

Reply:
do the same in goldplan